web/src/components/core/FileViewer.test.jsx

```diff
@@ -28,19 +28,19 @@ import cockpit from "../../lib/cockpit";
 
 jest.mock("../../lib/cockpit");
 
-const readFn = jest.fn((arg) => new Promise(jest.fn()));
+const readFn = jest.fn(() => new Promise(jest.fn()));
 
 const fileFn = jest.fn();
 fileFn.mockImplementation(() => {
   return {
     read: readFn
-  }
+  };
 });
 
 cockpit.file.mockImplementation(fileFn);
 
 // testing data
-const file_name = "/testfile"
+const file_name = "/testfile";
```

Comment:
Just a reminder for myself: The same than https://github.com/yast/d-installer/pull/418#pullrequestreview-1289770888